### PR TITLE
Resolve sniffer's errors and warning in src/admin-views/

### PR DESCRIPTION
**Pull request by pattihis**
### 🗒️ Description

Follow-up [to this PR](https://github.com/the-events-calendar/tribe-common/pull/2121) to resolve code sniffer's errors and warnings.

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

### CODE REVIEW

**Review by stratease — APPROVED** [review]
Seems ok. Let's get a second approve this is a huge PR, not confident I didn't miss something.

**Review by bordoni — APPROVED** [review]

File: src/admin-views/troubleshooting/ea-status/server-connection.php [hunk 1]
```diff
@@ -51,5 +51,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes;  // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```
Comment (bordoni) [inline src/admin-views/troubleshooting/ea-status/server-connection.php:54]: We dont need to do this again. there was a comment about how the escaping is handled above. maybe add the ignore for PHPCS.

File: src/admin-views/troubleshooting/ea-status/scheduler-status.php [hunk 2]
```diff
@@ -29,5 +28,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes;  // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```
Comment (bordoni) [inline src/admin-views/troubleshooting/ea-status/scheduler-status.php:31]: This was handled above. Not need to clean again, add the PHPCS ignore

File: src/admin-views/troubleshooting/ea-status/meetup.php [hunk 3]
```diff
@@ -35,5 +34,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes;  // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```
Comment (bordoni) [inline src/admin-views/troubleshooting/ea-status/meetup.php:37]: This was handled above. Not need to clean again, add the PHPCS ignore

File: src/admin-views/troubleshooting/ea-status/license-key.php [hunk 4]
```diff
@@ -50,5 +49,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes; // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```
Comment (bordoni) [inline src/admin-views/troubleshooting/ea-status/license-key.php:52]: This was handled above. Not need to clean again, add the PHPCS ignore

File: src/admin-views/troubleshooting/ea-status/eventbrite.php [hunk 5]
```diff
@@ -44,5 +44,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes;  // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```
Comment (bordoni) [inline src/admin-views/troubleshooting/ea-status/eventbrite.php:47]: This was handled above. Not need to clean again, add the PHPCS ignore

File: src/admin-views/troubleshooting/ea-status/current-usage.php [hunk 6]
```diff
@@ -38,5 +38,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes;  // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```
Comment (bordoni) [inline src/admin-views/troubleshooting/ea-status/current-usage.php:41]: This was handled above. Not need to clean again, add the PHPCS ignore

File: src/admin-views/troubleshooting/ea-status/current-status.php [hunk 7]
```diff
@@ -37,5 +35,5 @@
 		/>
 		<?php echo esc_html( $message ); ?>
 	</td>
-	<td><?php echo $notes;  // Escaping handled above. ?></td>
-</tr>
\ No newline at end of file
+	<td><?php echo wp_kses_post( $notes ); ?></td>
```
Comment (bordoni) [inline src/admin-views/troubleshooting/ea-status/current-status.php:38]: This was handled above. Not need to clean again, add the PHPCS ignore

File: src/admin-views/troubleshooting/common-issues.php [hunk 8]
```diff
@@ -31,10 +30,10 @@
 				<div class="tribe-events-admin-faq__answer">
 					<?php
 						$label = '<a href="' . esc_url( $common_issue['link'] ) . '" target="_blank" rel="noopener noreferrer">' . esc_html( $common_issue['link_label'] ) . '</a>';
-						echo sprintf( $common_issue['solution'], $label );
+						printf( esc_html( $common_issue['solution'] ), wp_kses_post( $label ) );
```
Comment (bordoni) [inline src/admin-views/troubleshooting/common-issues.php:33]: There is no need to `wp_kses_post` here, its too expensive to keep running this all the time.

File: src/admin-views/troubleshooting.php [hunk 9]
```diff
@@ -1,55 +1,59 @@
...
-					.closest( '.tribe-events-admin__issues-found-card' )
-					.find( '.tribe-events-admin__issues-found-card-description' )
-					.slideDown( 200 );
+					.addClass('active')
+					.closest('.tribe-events-admin__issues-found-card')
+					.find('.tribe-events-admin__issues-found-card-description')
+					.slideDown(200);
 			}
-		} );
+		});
```
Comment (bordoni) [inline src/admin-views/troubleshooting.php:58]: All of these changes are not what we want. Removed a bunch of spaces.

**Review by bordoni — CHANGES_REQUESTED** [review]

**Review by bordoni — DISMISSED** [review]

**Review by bordoni — APPROVED** [review]

**Pull request merged**